docs/example-scenario/ai/movie-recommendations.md

```diff
@@ -0,0 +1,131 @@
+
+5. A trained model is preserved on Azure Cosmos DB, which can then be applied for recommending the top *k* movies for a given user.
+
+6. The model is then deployed onto a web or app service by using Azure Container Instances or Azure Kubernetes Service.
+
+For an in-depth guide to building and scaling a recommender service, see [Build a real-time recommendation API on Azure][ref-arch].
+
+### Components
+
+* [Data Science Virtual Machine][dsvm] (DSVM) is an Azure virtual machine with deep learning frameworks and tools for machine learning and data science. The DSVM has a standalone Spark environment that can be used to run ALS.
+
+* [Azure Blob storage][blob] stores the dataset for movie recommendations.
+
+* [Azure Machine Learning service][mls] is used to accelerate the building, managing, and deploying of machine learning models.
+
+* [Azure Cosmos DB][cosmosdb] enables globally distributed and multi-model database storage.
+
+* [Azure Container Instances][aci] is used to deploy the trained models to web or app services, optionally using [Azure Kubernetes Service][aks].
+
+### Alternatives
+
+[Azure Databricks][databricks] is a managed Spark cluster where model training and evaluating is performed. You can set up a managed Spark environment in minutes, and [autoscale][autoscale] up and down to help reduce the resources and costs associated with scaling clusters manually. Another resource-saving option is to configure inactive [clusters][clusters] to terminate automatically.
+
+## Considerations
+
+### Availability
+
+Machine-learning-built apps are split into two resource components: resources for training, and resources for serving. Resources required for training generally do not need  high availability, as live production requests do not directly hit these resources. Resources required for serving need to have high availability to serve customer requests.
+
+For training, the DSVM is available in [multiple regions][regions] around the globe and meets the [service level agreement][sla] (SLA) for virtual machines. For serving, Azure Kubernetes Service provides a [highly available][ha] infrastructure. Agent nodes also follow the [SLA][sla-aks] for virtual machines.
+
+### Scalability
+
+If you have a large data size, you can scale your DSVM to shorten training time. You can scale a VM up or down by changing the [VM size][vm-size]. Choose a memory size large enough to fit your dataset in-memory and a higher vCPU count in order to decrease the amount of time that training takes.
+
+### Security
+
+This scenario can use Azure Active Directory to authenticate users for [access to the DSVM][dsvm-id], which contains your code, models, and (in-memory) data. Data is stored in Azure Storage prior to being loaded on a DSVM, where it is automatically encrypted using [Storage Service Encryption][storage-security]. Permissions can be managed via Azure Active Directory authentication or role-based access control.

```

Comment:
>Permissions can be managed

What permissions does this refer to?  Azure RBAC is a resource manager concept. It might help to separate out authN/authZ as it relates to: devops, model training, and model consuming